app/controllers/configuration_controller.rb

```diff
@@ -784,6 +785,7 @@ def get_form_vars
     when "ui_4"                                               # Visual Settings tab
       @edit[:new][:display][:compare] = params[:display][:compare] if !params[:display].nil? && !params[:display][:compare].nil?
       @edit[:new][:display][:drift] = params[:display][:drift] if !params[:display].nil? && !params[:display][:drift].nil?
+      @edit[:timeprofile_id] = TimeProfile.find(@edit[:timeprofile_id]) if @edit[:timeprofile_id].present?
```

Comment:
@ZitaNemeckova looks like you don't need this line, this is saving TimeProfile ActiveRecord in ``` @edit[:timeprofile_id] ```